service/internal/logger/audit/constants.go

```diff
@@ -1,5 +1,18 @@
 package audit
 
+import (
+	"encoding/json"
+)
+
+type RequestHeader string
+
+const (
+	UserAgentHeaderKey RequestHeader = "user-agent"
+	RequestIDHeaderKey RequestHeader = "x-request-id"
+	RequestIPHeaderKey RequestHeader = "x-forwarded-request-ip"
+	ActorIDHeaderKey   RequestHeader = "x-forwarded-actor-id"
+)
```

Comment:
if there is no circular dependencies, then use sdk constants
`sdkAudit "github.com/opentdf/platform/sdk/audit"`

Reply:
these actually weren't being referenced anywhere as i had migrated everything off of them so I just deleted them